.travis.yml

```diff
@@ -95,13 +89,27 @@ install:
 
   # Update the package list and install the Cloud SDK
   - sudo apt-get -qq update
-  - sudo apt-get install google-cloud-sdk -y
+  - sudo apt-get -qq install google-cloud-sdk -y
+
+  # Installing Azure command-line client here as used both in cleaning and cron_job scripts
+  - echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ wheezy main" | sudo tee /etc/apt/sources.list.d/azure-cli.list
+  - sudo apt-key adv --keyserver packages.microsoft.com --recv-keys 52E16F86FEE04B979B07E28DB02C46DF417A0893
+  - sudo apt-get -qq install apt-transport-https -y
+  - sudo apt-get -qq update && sudo apt-get -qq install azure-cli -y
 
 before_script:
-  # Get current_version, variable common to all three platforms
-  - source ./bin/get_current_version.sh
```

Comment:
This part has been modified according to the new logic of the cron_script for security updates.